### ISSUE TITLE
Use inputs as outputs

### ISSUE THREAD

**Issue by Euphorbium** [VCVRack/Rack#306]
It should be possible to take a cable connected to an input with a modifier key, and essentially splice it, routing the signal further with another cable. This would increase production speed, and reduce cable clutter onscreen.

**Comment by Euphorbium** [comment]
Is the idea clear?
After using an input as multiple outputs, once you take that plug without a modifier, all other cables should be removed, unless another modifier key was used (maybe shift?).

**Comment by maricn** [comment]
If i got it right - Basically, `Wire`s could connect single `Output` to vector of `Input`s instead of enumerated single `Module`'s `Input`? Step through wire would cascade signal fire.
Regarding stacking, it's UX, but I'd maintain the notion of top wire (last added wire to the stack), so only with modifier would be able to remove all the wires at once. Or, ofc, by removing one by one until getting the input wire out..

**Comment by AndrewBelt** [comment]
Yeah, that'd be cool, but before I start adding cable enhancements, I need to think over the whole design space of what to do with combinations of key commands (Shift, Ctrl/Cmd), dragging inputs/outputs, and dropping on inputs/outputs. I'll create an issue after v1.0 that organizes all these "cable thoughts".

**Comment by WIZARDISHUNGRY** [comment]
@AndrewBelt I find myself trying to use the quartz composer modifiers when dragging cables, perhaps  that's a decent program to copy.
